pages/_app.tsx

```diff
@@ -2,4 +2,12 @@ import App from "next/app";
 
 import "normalize.css";
 
+import Store from "../store/Store";
```

Comment:
I know I named this, but I think it's ultimately confusing. Let's rename this concept to `storage/Storage` or `database/Database` (on the fence about which).

Reply:
Still needs attention.